src/email_templates/manager_approval_requested/html.pug

```diff
@@ -0,0 +1,9 @@
+p Hello,    
+|
+|
+p Activity Report #{displayId} was submitted for your review.
```

Comment:
Should these be hyperlinked too like in the "Changes requested by manager?"

Reply:
I mean the `#{displayId}` bit

Reply:
It is linked right below.
```
p Access this report in the TTA Smart Hub:  
  a(href=reportPath)
|
| Best Regards,
| Your TTA Hub Team 
```